b/id.go

```diff
@@ -0,0 +1,85 @@
+package xidb
+
+import (
+	"database/sql/driver"
+	"fmt"
+	"time"
+
+	"github.com/rs/xid"
+)
+
+// ID represents a unique request id
+type ID struct {
+	xid.ID
```

Comment:
Wouldn't `type ID xid.ID` work better in this context?

Reply:
So, if you define the new type like this, I don't think you inherit its methods.
https://play.golang.org/p/e-8XEdbQ-tH
Or maybe there is something I don't see?

Reply:
You’re right. But do the wrapper need this merhod to work? We can imagine the wrapper only be used as an adapter when used with db. You always keep the naked version for other things.

It’s just a proposal. I’m also fine with the anonymous composition of you think it’s a better design.

Reply:
I can't see how that would work. If we think about a normal use case, like a struct defined with a xid:
type boat struct {
  ID xid.ID
}
Let's say I store this in mysql with a BINARY(12)
I'd probably need to redefine the Scan and Value for this struct as well if xidb doesn't do it, am I right?

Reply:
Ok I see. Again, I never used SQL with Go, so I don't have a clear view of how it would be used. Let's go with anonymous composition then. I still think it make sense to not repeat all the constructor methods in this package, but instead provide an easy way to wrap an existing xid.